doc/classes/ReflectionProbe.xml

```diff
@@ -4,8 +4,9 @@
 		Captures its surroundings to create reflections.
 	</brief_description>
 	<description>
-		Capture its surroundings as a dual parabolid image, and stores versions of it with increasing levels of blur to simulate different material roughnesses.
+		Capture its surroundings as a dual paraboloid image, and stores versions of it with increasing levels of blur to simulate different material roughnesses.
 		The [ReflectionProbe] is used to create high-quality reflections at the cost of performance. It can be combined with [GIProbe]s and Screen Space Reflections to achieve high quality reflections. [ReflectionProbe]s render all objects within their [member cull_mask], so updating them can be quite expensive. It is best to update them once with the important static objects and then leave them.
+		Note: By default Godot will only render 16 reflection probes. If you need more increase Atlas Subdivs, This setting can be found in Project Settings -> Quality -> Reflections 
```

Comment:
You can actually link directly to project settings using the suggested syntax below. :)

```suggestion
		Note: By default Godot will only render 16 reflection probes. If you need more, increase the number of atlas subdivisions. This setting can be found in [member ProjectSettings.rendering/quality/reflections/atlas_subdiv].
```